data/applications/openvino.yaml

```diff
@@ -0,0 +1,14 @@
+metadata:
+  name: openvino
+spec:
+  displayName: OpenVINO
+  provider: Intel
+  description: |-
+    OpenVINO toolkit facilitates the optimization of a deep learning model from a framework and deployment using an inference engine onto Intel hardware.
+  kfdefApplications: ['jupyterhub', 'notebook-images']
```

Comment:
I'm not sure if we'd add stuff here to install the operator by default or add images?

Reply:
Only if installing the KfDef for `jupyterhub` or `notebook-images` indicates that `OpenVino` is installed should this be set.